src/main/java/at/hannibal2/skyhanni/features/bazaar/BazaarDataHolder.kt

```diff
@@ -28,28 +28,21 @@ class BazaarDataHolder {
 
     private fun loadNpcPrices(): MutableMap<NEUInternalName, Double> {
         val list = mutableMapOf<NEUInternalName, Double>()
-        val itemsData = APIUtil.getJSONResponse("https://api.hypixel.net/v2/resources/skyblock/items")
-
+        val apiResponse = APIUtil.getJSONResponse("https://api.hypixel.net/v2/resources/skyblock/items")
         try {
-            val motesPrice = mutableMapOf<NEUInternalName, Double>()
-            for (item in itemsData.getAsJsonArray("items")) {
-                val itemData = item.asJsonObject
+            val itemsData = ConfigManager.gson.fromJson(apiResponse, SkyblockItemsDataJson::class.java)
```

Comment:
saw this function in JsonUtils, improves readability
```suggestion
            val itemsData = ConfigManager.gson.fromJson<SkyblockItemsDataJson>(apiResponse)
```

Reply:
Done, ty!